eldritch/items/unique.py

```diff
@@ -350,3 +350,97 @@ def get_interrupt(self, event, owner, state):
     if isinstance(event, events.CityMovement) and event.character == owner:
       return events.ChangeMovementPoints(owner, 3)
     return None
+
+# Tomes
+
+
+class TibetanTome(Tome):
+  def __init__(self, idx):
+    super().__init__("Tibetan Tome", idx, "unique", 3, 2)
+    self.max_tokens["stamina"] = 2
+
+  def read_event(self, owner):
+    check = events.Check(owner, "lore", -1)
+    success = events.Sequence([
+        events.Draw(owner, "spells", 1),
+        events.Loss(owner, {"sanity": 1}, source=self),
```

Comment:
I would recommend putting the token before the sanity loss - that way, if they go insane, they still put the token on the tome.